src/content/getusermedia/resolution/js/main.js

```diff
@@ -103,7 +103,8 @@ video.onplay = function() {
 function getMedia(constraints) {
   if (stream) {
     video.src = null;
-    stream.stop();
+    stream.getAudioTracks().forEach(function(track) { track.stop(); });
```

Comment:
possible helper function for adapter? seems like this will be used in multiple places